app/services/flow/flows/provider_merits.rb

```diff
@@ -14,11 +14,13 @@ class ProviderMerits < FlowSteps
         },
         respondents: {
           path: ->(application) { urls.providers_legal_aid_application_respondent_path(application) },
-          forward: :client_received_legal_helps,
+          forward: :statement_of_cases,
           check_answers: :check_merits_answers
         },
         client_received_legal_helps: {
+          # :nocov:
```

Comment:
Sorry to be a pain - but could you also add a `TODO:` comment stating when the `nocov` should be removed.

Reply:
No problemo 👍